homeassistant/components/device_tracker/mercedesme.py

```diff
@@ -49,23 +49,22 @@ def __init__(self, hass, config, see, data):
     def update_info(self, now=None):
         """Update the device info."""
         for device in self.data.cars:
-            _LOGGER.debug("Updating %s", device["vin"])
-            location = self.data.get_location(device["vin"])
-            if location is None:
-                return False
-            dev_id = device["vin"]
-            name = device["license"]
-
-            lat = location['positionLat']['value']
-            lon = location['positionLong']['value']
-            attrs = {
-                'trackr_id': dev_id,
-                'id': dev_id,
-                'name': name
-            }
-            self.see(
-                dev_id=dev_id, host_name=name,
-                gps=(lat, lon), attributes=attrs
-            )
+            if device['services'].get('VEHICLE_FINDER', False):
```

Comment:
Make this a guard clause instead by inverting the check and continue the iteration if the check is true. Then you can outdent the following block.